plugins/paymethod/paypal/locale/cs_CZ/locale.xml

```diff
@@ -4,8 +4,8 @@
 <!--
   * plugins/paymethod/paypal/locale/cs_CZ/locale.xml
   *
-  * Copyright (c) 2014-2016 Simon Fraser University Library
-  * Copyright (c) 2006-2014 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
+  * Copyright (c) 2013-2016 Simon Fraser University Library
+  * Copyright (c) 2006-2009 Gunther Eysenbach, Juan Pablo Alperin, MJ Suhonos
```

Comment:
To clarify: no other file but this has a copyright longer than 2009 for the non-team contributors. This is what the en_US original file looks like. Please decide - I can revert this, but I don't think it's necessary.